### PR TITLE
Fix original error being overwritten before returned

### PR DESCRIPTION
**Pull request by aliscott**

### CODE REVIEW

**Comment by brendandburns** [comment]
/lgtm

LGTM from the bulk LGTM tool

**Comment by brendandburns** [comment]
Apologies in advance, I'm going to use this PR for some testing of the bulk-lgtm tool...  Please ignore the noise.

File: pkg/volume/glusterfs/glusterfs.go [hunk 1]
```diff
@@ -750,9 +750,9 @@ func (p *glusterfsVolumeProvisioner) CreateVolume(gid int) (r *v1.GlusterfsVolum
 	endpoint, service, err := p.createEndpointService(epNamespace, epServiceName, dynamicHostIps, p.options.PVC.Name)
 	if err != nil {
 		glog.Errorf("glusterfs: failed to create endpoint/service: %v", err)
-		err = cli.VolumeDelete(volume.Id)
-		if err != nil {
-			glog.Errorf("glusterfs: error when deleting the volume :%v , manual deletion required", err)
+		delete_err = cli.VolumeDelete(volume.Id)
```
Comment (brendandburns) [inline pkg/volume/glusterfs/glusterfs.go]: you need `delete_err := ...` (colon-equals)

Comment (aliscott) [inline pkg/volume/glusterfs/glusterfs.go]: Thanks, fixed

**Comment by brendandburns** [comment]
@k8s-bot test this please

**Comment by brendandburns** [comment]
/lgtm

/release-note-none

LGTM + release-note-none from the bulk LGTM tool

**Pull request merged**